src/main/groovy/com/netgrif/application/engine/startup/DashboardRunner.groovy

```diff
@@ -0,0 +1,61 @@
+package com.netgrif.application.engine.startup
+
+import com.netgrif.application.engine.petrinet.domain.PetriNet
+import com.netgrif.application.engine.petrinet.domain.VersionType
+import com.netgrif.application.engine.petrinet.service.interfaces.IPetriNetService
+import groovy.util.logging.Slf4j
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.stereotype.Component
+
+@Slf4j
+@Component
+@ConditionalOnProperty(value = "nae.dashboard.enabled", matchIfMissing = false)
+class DashboardRunner extends AbstractOrderedCommandLineRunner {
+
+    @Autowired
+    private IPetriNetService petriNetService
+
+    @Autowired
+    private ImportHelper helper
+
+    @Autowired
+    private SystemUserRunner systemCreator
+
+    public static final String DASHBOARD_NET_IDENTIFIER = "dashboard"
+    private static final String DASHBOARD_FILE_NAME = "engine-processes/dashboard.xml"
```

Comment:
This could be `public`, it is already static final so I see no harm in being public.